cfgov/jinja2/v1/_includes/molecules/image-inset.html

```diff
@@ -0,0 +1,74 @@
+{# ==========================================================================
+
+    Image Inset
+
+    ==========================================================================
+
+    Description:
+
+    Creates image inset markup when given:
+
+    value:                   Object defined from a StreamField block.
+
+    value.heading:           The heading of the image inset.
+
+    value.image:             A basic image object to insert.
+
+    value.image.upload:      The image object.
+
+    value.image.alt:         String for alt text for screenreaders.
+
+    value.paragraph:         String for text below the image.
+
+    value.is_image_left:     Boolean indicating whether to float the image
+                             to the left.
+
+    value.image_width:       String representing the image width.
+                             Possible values: (150, 270, 300).
+                             Defaults to 270.
+
+   ========================================================================== #}
+
+{# Image sizes are doubled for retina displays. Images created via Wagtail
+   are never stretched or upscaled and maintain their aspect ratio. #}
+{% set image_inset = image(value.image.upload, 'width-' ~ value.image_width
+                     | int * 2) %}
+{% set image_mobile = image(value.image.upload, 'width-1200') %}
+<div class="m-inset
+            m-inset__image
+            {{ 'm-inset__image__left' if value.is_image_left else '' }}
+            {{ 'm-inset__image__' ~ ( value.image_width
+               if value.image_width else '270' ) }}
+            {{ 'm-inset__image__portrait'
+               if image_inset.height > image_inset.width else 'landscape' }}">
+    {% if value.heading %}
+    <h2 class="header-slug">
+        <span class="header-slug_inner">
+            {{ value.heading }}
+        </span>
+    </h2>
+    {% endif %}
+    <div class="m-inset_image-container">
+        {% if value.is_image_decorative %}
```

Comment:
There shouldn't be a difference, in an ideal world. All of these examples _should_ be considered decorative. The inconsistent implementation comes down to workflow constraints or lack of knowledge around this issue.

The image on the blog post _should_ have been decorative, but this is pretty difficult to do correctly in a CMS (i.e., making it a CSS background image). Fortunately this image _does_ do the next best thing, which is having an empty `alt` attribute, which I believe means that most screenreaders will skip over it.

The isocons on http://www.consumerfinance.gov/your-story/ are also decorative, and should have probably been coded as background images, since this page was built by developers.